src/test/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessorTest.java

```diff
@@ -221,6 +245,8 @@ void GIVEN_one_component_WHEN_one_file_less_than_max_THEN_reads_entire_file(
             assertEquals(12, localDate.getMonth().getValue());
             assertEquals(17, localDate.getDayOfMonth());
         }
+        chan.close();
+        assertFalse(chan.isOpen());
```

Comment:
You are closing the channel on the previous line so this assertion is not really useful